packages/storefront-sdk/src/client/index.ts

```diff
@@ -110,6 +113,10 @@ export class StorefrontClient {
 		};
 	}
 
+	/**
+	 * Turn preview mode on or off by setting a `previewToken` to the SDK config. The `previewToken` can be found in the dashboard .
+	 * @param setConfigParams an object containing the `previewToken` property, supply a dashboard preview token to turn preview mode on and `null` or an empty object to turn it off.
```

Comment:
A few suggestions here:
- Some rephrasing for clarity
- I don't think `and `null` or an empty object to turn it off.` is correct - the way that `setConfig` is implemented, `null`, `undefined`, or an empty string should disable preview mode, but an empty object would not.
- I think this could benefit from [`@example`](https://tsdoc.org/pages/tags/example/)s

```suggestion
	 * Turn preview mode on or off on-the-fly by setting a `previewToken` in the SDK config. The `previewToken` can be generated in the Nacelle Dashboard.
	 * @param setConfigParams an object containing the `previewToken` property. Providing a value of `null`, `undefined`, or an empty string will disable preview mode.
	 *
	 * @example
	 * Enable preview mode by providing a `previewToken`:
	 * ```
	 * client.setConfig({ previewToken: '<my-nacelle-preview-token>' });
	 * ```
	 *
	 * @example
	 * Disable preview mode by providing `previewToken: null`:
	 * ```
	 * client.setConfig({ previewToken: null });
	 * ```
```

Example output:

![VSCode in-editor feedback with TSDoc information when using the setConfig method](https://user-images.githubusercontent.com/5732000/212425491-e7406399-ca8d-49d6-8522-73ef51c96608.png)